### ISSUE TITLE
Start submission IDs with another number than one and enable change of submission ID 

### ISSUE THREAD

**Issue by user34ngrau** [agroportal/project-management#511]
From discussion with Clément about the possibility to influence the "ID submission" of an ontology. Possibility to start with a latest version first and then can have the option to upload oldest version after, if they existe.
<img width="713" alt="image" src="https://github.com/agroportal/project-management/assets/135241416/71caa7df-f725-4517-bc4f-13afd79c12fd">

**Comment by jonquet** [comment]
In fact, at its simplest it would be:

- we change the number of a first submission to 100 instead of 1 => certainly easy
- we create the ability to select the number of a submission (admin only) and get an error if number is taken => hard 

This will allow us to load submissions in any order (relative to versions).

The main use case is when we load an ontology for the first time but we do know that there are multiple versions of this ontology that we would like to also archive in AgroPortal. Starting at 100 will allow us to "come back later" to use the number 1 to 99 to store "previous versions".

**Comment by syphax-bouazzouni** [comment]
> we change the number of a first submission to 100 instead of 1 => certainly easy

Changing the submission ID number, means, reloading all the ontology data into a new graph, as submission data are saved in the submission ID graph, e.g the data for E-PHY is saved in the graph  `"https://data.agroportal.lirmm.fr/ontologies/E-PHY/submissions/2`)

In addition, you will need to migrate all the metadata that have as subject the old submission ID to the new one, e.g for E-PHY it means fetching all the tripes that matches  `<https://data.agroportal.lirmm.fr/ontologies/E-PHY/submissions/2> ?p ?v` and migrate them to the new ID

In summary, changing a submission ID, means preparing the ontology with new ID and migrating all the metadata.

**Comment by saubin78** [comment]
Ez-publish propose un système intéressant pour ordonner des contenus
On peut mettre des nombres positifs ou négatifs et on peut changer le numéro (ce n'est pas un identifiant)
![image](https://github.com/agroportal/project-management/assets/18008698/763c3c27-1285-4a08-a4e7-12679c7c5966)

https://alain-sahli.developpez.com/tutoriels/php/ez/publish/configurer-site/images/7_update_priority_levels.png

**Comment by jonquet** [comment]
Est-ce que ce on pourrait réfléchir a investir plutôt sur un script qui serait capable de reconstruire complètement l'historique d'une ontologie... ?

**Comment by imadbourouche** [comment]
to refine
